src/Hydration.js

```diff
@@ -0,0 +1,38 @@
+class Hydration {
+  constructor(userID, hydrationData) {
+    this.userID = userID;
+    this.userData = hydrationData.filter(element => element.userID === userID);
+  }
+
+  calculateToalOunces() {
+    const total = this.userData.reduce(function(sum,current) {
+      return sum + current.numOunces
+  }, 0);
+  return total;
+  }
+
+  calculateDailyOunces(date) {
+    let thisDay;
+ 
+    thisDay = this.userData.filter(element => element.date === date);
```

Comment:
We could get rid of line 15 and declare `thisDay` on line 17 to dry this up a tidbit